.github/ISSUE_TEMPLATE/02_feature.yml

```diff
@@ -6,7 +6,7 @@ body:
       value: >
         Please read this brief portion of documentation before going any
         further:
-        http://flake8.pycqa.org/en/latest/internal/contributing.html#filing-a-bug
```

Comment:
ah we still want them to read everything here because it requests that they search the issue tracker

Reply:
My bad. I missed the anchor change

Reply:
okay my bad -- i thought the anchor was wrong as there is an extra paragraph for feature requests but i get your point @asottile.